### PR TITLE
Create cross-device_app_configuration

### PR DESCRIPTION
**Pull request by jennifershowe**
additional conceptual topic which includes instructions for configuring cross-device app profile in new Dev Center experience as well as externally.

### CODE REVIEW

File: concepts/cross-device_app_configuration [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Delight users with cross-device apps, powered by Project Rome
```
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Please rename the file to use all hyphens.

File: concepts/cross-device_app_configuration [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+### Step 2: Collect your platform-specific application IDs
+Collect the platform-specific application IDs for each application and platform which will use the Activity Feed and/or Device Relay API. 
+You'll need to collect each of the platform specific application IDs in order to associate them to your cross-device app identity. Using Windows Dev Center, you'll be able to select from Windows universal apps associated to your developer account, but you'll need to manually provide application ids for any of your win32 apps and/or apps available on iOS & Android and identify the primary URL for your web applications. If you're using an externally hosted JSON file you'll assemble these identifiers into the specified format outlined below. You can associate up to 10 ids per platform. 
+
+#*Where do I find these ids?*#
```
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Please fix the #s here.

File: concepts/cross-device_app_configuration (hunk 1 above)
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Can we use a more actionable title? Note that the first words are the most important for SEO; "Delight users" isn't really what developers will be searching for. Something like "Build cross-device apps, powered by Project Rome".

File: concepts/cross-device_app_configuration [hunk 3]
```diff
@@ -0,0 +1,192 @@
+# Delight users with cross-device apps, powered by Project Rome 
+
+Project Rome aims to help developers build experiences that cross devices and platforms seamlessly, reducing friction for users and helping to drive app engagement. In order for applications to share data across devices & platforms using Project Rome APIs an app developer can configure a cross-device app which includes information about your platform-specific apps.
```
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Revise the first sentence: You can use Project Rome to build experiences that cross devices and platforms seamlessly, reducing friction for users and helping to drive app engagement. 

The second sentence is unclear - please revise. Note that in content we refer to the developer audience as "you", so please change to second person.

File: concepts/cross-device_app_configuration [hunk 4]
```diff
@@ -0,0 +1,192 @@
+# Delight users with cross-device apps, powered by Project Rome 
+
+Project Rome aims to help developers build experiences that cross devices and platforms seamlessly, reducing friction for users and helping to drive app engagement. In order for applications to share data across devices & platforms using Project Rome APIs an app developer can configure a cross-device app which includes information about your platform-specific apps. 
+
+A cross-device app is required to take advantage of the following capabilities powered by Project Rome:
```
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Is "required" the right word here? Do you mean something like "A cross-device app is powered by the following capabilities in Project Rome"?

File: concepts/cross-device_app_configuration [hunk 5]
```diff
@@ -0,0 +1,192 @@
+# Delight users with cross-device apps, powered by Project Rome 
+
+Project Rome aims to help developers build experiences that cross devices and platforms seamlessly, reducing friction for users and helping to drive app engagement. In order for applications to share data across devices & platforms using Project Rome APIs an app developer can configure a cross-device app which includes information about your platform-specific apps. 
+
+A cross-device app is required to take advantage of the following capabilities powered by Project Rome: 
+
+	1. Use the activity feed API on MSGraph
```
Comment (Lauragra) [inline concepts/cross-device_app_configuration]: Change "MSGraph" to Microsoft Graph.

File: concepts/cross-device_app_configuration (hunk 4 above)
Comment (jennifershowe) [reply concepts/cross-device_app_configuration]: Required is correct. This is literally required configuration to take advantage of the listed capabilities.

File: concepts/cross-device_app_configuration (hunk 1 above)
Comment (jennifershowe) [reply concepts/cross-device_app_configuration]: Done

Comment (jennifershowe) [reply concepts/cross-device_app_configuration]: Done

**Pull request merged**